sympy/physics/mechanics/joint.py

```diff
@@ -543,3 +533,217 @@ def _set_linear_velocity(self):
+    parent : Body
+        The parent body of joint.
+    child : Body
+        The child body of joint.
+    coordinates: dynamicsymbol, optional
+        Generalized coordinates of the joint.
+    speeds : dynamicsymbol, optional
+        Generalized speeds of joint.
+    parent_joint_pos : Vector, optional
+        Vector from the parent body's mass center to the point where the parent
+        and child are connected. The default value is the zero vector.
+    child_joint_pos : Vector, optional
+        Vector from the child body's mass center to the point where the parent
+        and child are connected. The default value is the zero vector.
+    parent_axis : Vector, optional
+        Axis fixed in the parent body which aligns with an axis fixed in the
+        child body. The default is x axis in parent's reference frame.
+    child_axis : Vector, optional
+        Axis fixed in the child body which aligns with an axis fixed in the
+        parent body. The default is x axis in child's reference frame.
+
+    Attributes
+    ==========
+
+    name : string
+        The joint's name.
+    parent : Body
+        The joint's parent body.
+    child : Body
+        The joint's child body.
+    coordinates : list
+        List of the joint's generalized coordinates.
+    speeds : list
+        List of the joint's generalized speeds.
+    parent_point : Point
+        The point fixed in the parent body that represents the joint.
+    child_point : Point
+        The point fixed in the child body that represents the joint.
+    parent_axis : Vector
+        The axis fixed in the parent frame that represents the joint.
+    child_axis : Vector
+        The axis fixed in the child frame that represents the joint.
+    kdes : list
+        Kinematical differential equations of the joint.
+
+    Examples
+    =========
+
+    A single prismatic joint is created from two bodies and has the following basic
+    attributes:
+
+    >>> from sympy.physics.mechanics import Body, PrismaticJoint
+    >>> parent = Body('P')
+    >>> parent
+    P
+    >>> child = Body('C')
+    >>> child
+    C
+    >>> joint = PrismaticJoint('PC', parent, child)
+    >>> joint
+    PrismaticJoint: PC  parent: P  child: C
+    >>> joint.name
+    'PC'
+    >>> joint.parent
+    P
+    >>> joint.child
+    C
+    >>> joint.parent_point
+    PC_P_joint
+    >>> joint.child_point
+    PC_C_joint
+    >>> joint.parent_axis
+    P_frame.x
+    >>> joint.child_axis
+    C_frame.x
+    >>> joint.coordinates
+    [x_PC(t)]
+    >>> joint.speeds
+    [v_PC(t)]
+    >>> joint.child.frame.ang_vel_in(joint.parent.frame)
+    0
+    >>> joint.child.frame.dcm(joint.parent.frame)
+    Matrix([
+    [1, 0, 0],
+    [0, 1, 0],
+    [0, 0, 1]])
+    >>> joint.child_point.pos_from(joint.parent_point)
+    x_PC(t)*P_frame.x
+
+    To further demonstrate the use of the prismatic joint, the kinematics of n-link
+    pendulum(n=2) about the X axis of each connected body can be created as follows.
```

Comment:
You can't create a pendulum with a sliding joint. You can create this https://github.com/pydy/pydy/blob/master/pydy/models.py#L16 (but with out springs and dampers in between the masses). This example should be two masses sliding, one moving relative to a fixed body and the other relative to the moving body.

Reply:
I'll change that.